### PR TITLE
Add new data

### PR DESCRIPTION
**Pull request by zmeers**
Hi Rob,

I hope you're doing well - it has been a while. I'm pushing an update with new data to `ggparliament`.

I think there are some smaller stylistic choices that can be made too - e.g. `group_nest()` for splitting, applying, etc., but I don't have time to do at the moment... It's on my TODO list.

Thanks,
Zoe

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -170,10 +152,19 @@ senate <- ggplot(us_senate, aes(x, y, colour = party_long)) +
...
+## Error in loadNamespace(name): there is no package called 'ggrepel'
+```
+
+```r
 senate 
 ```
 
-![plot of chunk unnamed-chunk-7](figure/unnamed-chunk-7-1.png)
+```
+## Error in eval(expr, envir, enclos): object 'senate' not found
```
Comment (RobWHickman) [inline README.md]: should these (+line 156) be showing errors?

**Review by RobWHickman — APPROVED** [review]
LGTM pending question about errors in the README

**Comment by RobWHickman** [comment]
(also when I get time I'll take a look at the underlying code and refactor some- probably not for a few months given current job- I might also have to get in touch for some work Im doing on australian elections in the near future :) Hope you're well!)

File: README.md (hunk 1 above)
Comment (zmeers) [reply README.md]: Fixed - new laptop, hadn't installed `ggrepel`. Thanks.

**Comment by zmeers** [comment]
> (also when I get time I'll take a look at the underlying code and refactor some- probably not for a few months given current job- I might also have to get in touch for some work Im doing on australian elections in the near future :) Hope you're well!)

SGTM!

**Pull request merged**